p5/sketch/userspace.py

```diff
@@ -311,9 +311,7 @@ def save(filename='screen.png'):
     :type filename: str
 
     """
-    # TODO: images saved using ``save()`` should *not* be numbered.
-    # --abhikpal (2018-08-14)
-    p5.sketch.screenshot(filename)
+    p5.sketch.screenshot(filename, numbered_fname=False)
```

Comment:
Does this setting matches the ones in Processing? 

Maybe you can add default attribute in the `def save(filename='screen.png')` as well

Reply:
Oh, I didn’t even consider what the setting was in Processing😅

I’ll make it consistent!